### PR TITLE
Generic stories for remaining core features

### PR DESCRIPTION
**Pull request by tmeasday**
Issue: following on from https://github.com/storybookjs/storybook/pull/19105

## What I did

- Add + remove remaining "core" stories from official SB.

### CODE REVIEW

File: code/lib/store/template/stories/rendering.stories.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+export const ChangeArgs = {
+  play: async ({ canvasElement, id }: PlayFunctionContext) => {
+    // const channel = globalThis.__STORYBOOK_ADDONS_CHANNEL__;
+    const button = await within(canvasElement).findByRole('button');
+    await button.focus();
+    await expect(button).toHaveFocus();
+
+    // // When we change the args to the button, it should not rerender
+    // await channel.emit('updateStoryArgs', { storyId: id, updatedArgs: { children: 'New Text' } });
```
Comment (shilman) [inline code/lib/store/template/stories/rendering.stories.ts:34]: why are these commented?

Comment (tmeasday) [reply code/lib/store/template/stories/rendering.stories.ts:34]: I comment them back in in https://github.com/storybookjs/storybook/pull/19125#pullrequestreview-1099066241

**Comment by shilman** [comment]
@tmeasday I updated from the base branch but CI is still failing. Can you please take a look?

**Review by shilman — APPROVED** [review]
🚀

**Pull request merged**